sdk_container/src/third_party/coreos-overlay/app-emulation/docker-cli/docker-cli-20.10.24.ebuild

```diff
@@ -6,7 +6,7 @@ GIT_COMMIT=e78084afe5
 EGO_PN="github.com/docker/cli"
 
 COREOS_GO_PACKAGE="${EGO_PN}"
-COREOS_GO_VERSION="go1.18"
+COREOS_GO_VERSION="go1.19"
```

Comment:
A fat comment here to always align this with the upstream build system would be nice

Reply:
E.g. point to https://github.com/docker/cli/blob/master/Dockerfile#L4

Reply:
and https://github.com/moby/moby/blob/master/Dockerfile#L3 etc.

Reply:
I'll add that to main